K2Bridge.Tests.End2End/flights/MSearch_Viz_Terms_OrderCount.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
generated using Kibana 7?

Reply:
The query was captured using MITMproxy using a Kibana 7 instance. This time I took the "raw" body so it should be exactly what Kibana sent.